packages/create-plugin/src/commands/update.command.ts

```diff
@@ -1,81 +1,80 @@
-import { TEXT, UDPATE_CONFIG } from '../constants.js';
-import { compileTemplateFiles, getTemplateData } from '../utils/utils.templates.js';
-import { confirmPrompt, selectPrompt, printMessage, printSuccessMessage } from '../utils/utils.console.js';
-import {
-  updatePackageJson,
-  hasNpmDependenciesToUpdate,
-  getPackageJsonUpdatesAsText,
-  updateNpmScripts,
-  writePackageManagerInPackageJson,
-} from '../utils/utils.npm.js';
-import { getPackageManagerWithFallback } from '../utils/utils.packageManager.js';
+import chalk from 'chalk';
+import { EXTRA_TEMPLATE_VARIABLES } from '../constants.js';
+import { updatePluginBuildConfig, isPluginDirectory } from '../utils/utils.plugin.js';
+import { getRuntimeVersion } from '../utils/utils.version.js';
+import { printRedBox, printBlueBox } from '../utils/utils.console.js';
+import { updatePackageJson, updateNpmScripts } from '../utils/utils.npm.js';
+import { isGitDirectory, isGitDirectoryClean } from '../utils/utils.git.js';
 
 export const update = async () => {
   try {
-    // 0. Warning
-    // ----------
-    printMessage(TEXT.updateCommandWarning);
+    if (!(await isGitDirectory())) {
+      printRedBox({
+        title: 'You are not inside a git directory',
+        content: `In order to proceed please run "git init" in the root of your project and commit your changes.\n
+(This check is necessary to make sure that the updates are easy to revert and don't mess with any changes you currently have.
+In case you want to proceed as is please use the ${chalk.bold('--force')} flag.)`,
+      });
 
-    // 1. Add / update configuration files
-    // -----------------------------------
-    if (await confirmPrompt(TEXT.updateConfigPrompt)) {
-      compileTemplateFiles(UDPATE_CONFIG.filesToOverride, getTemplateData());
-      printSuccessMessage(TEXT.overrideFilesSuccess);
-    } else {
-      printMessage(TEXT.overrideFilesAborted);
-      process.exit(0);
+      process.exit(1);
     }
 
-    // 2. Add / update dev dependencies inside the `package.json`
-    // (skipped automatically if there is nothing to update)
-    // ------------------------------------------------
-    if (hasNpmDependenciesToUpdate({ devOnly: true })) {
-      const updatableText = getPackageJsonUpdatesAsText({ ignoreGrafanaDependencies: true, devOnly: true });
+    if (!(await isGitDirectoryClean())) {
```

Comment:
Likewise, there's no logic to handle `--force`

Reply:
https://github.com/grafana/plugin-tools/pull/702#discussion_r1470784716

Reply:
Resolved in https://github.com/grafana/plugin-tools/pull/702/commits/e3720a6d75fb873ba700efd733be38dcc683e4cf.